From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Rudi Heitbaum <rudi@heitbaum.com>
Date: Sun, 2 Aug 2026 23:51:00 +1000
Subject: [PATCH] arm64: dts: imx8mq-phanbell: disable pcie, hdmi and dcss

Bring-up aid: turn everything off so the board can be brought back one
device at a time. Delete a block to re-enable that device.

Signed-off-by: Rudi Heitbaum <rudi@heitbaum.com>
---
 .../boot/dts/freescale/imx8mq-phanbell.dts    | 21 +++++++++++++++++++
 1 file changed, 21 insertions(+)

diff --git a/arch/arm64/boot/dts/freescale/imx8mq-phanbell.dts b/arch/arm64/boot/dts/freescale/imx8mq-phanbell.dts
index 2706e58..8e7f2b4 100644
--- a/arch/arm64/boot/dts/freescale/imx8mq-phanbell.dts
+++ b/arch/arm64/boot/dts/freescale/imx8mq-phanbell.dts
@@ -807,3 +807,24 @@
 		>;
 	};
 };
+
+/*
+ * Bring-up switches. These come last in the file, so they win over the
+ * status = "okay" set above. Delete a block to re-enable that device.
+ */
+
+&pcie0 {
+	status = "disabled";
+};
+
+&pcie1 {
+	status = "disabled";
+};
+
+&mhdp {
+	status = "disabled";
+};
+
+&dcss {
+	status = "disabled";
+};
